src/redux/skybrowser/skybrowserSlice.ts

```diff
@@ -0,0 +1,119 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+export interface SkyBrowserImage {
+  cartesianDirection: number[];
+  collection: string;
+  credits: string;
+  creditsUrl: string;
+  dec: number;
+  fov: number;
+  hasCelestialCoords: boolean;
+  identifier: string;
+  key: string;
+  name: string;
+  ra: number;
+  thumbnail: string;
+  url: string;
+}
+
+export interface SkyBrowserBrowser {
+  borderRadius: number;
+  cartesianDirection: number[];
+  color: number[];
+  dec: number;
+  displayCopies: object;
+  fov: number;
+  id: string;
+  isFacingCamera: boolean;
+  isUsingRae: boolean;
+  name: string;
+  opacities: number[];
+  ra: number;
+  ratio: number;
+  roll: number;
+  scale: number;
+  selectedImages: number[];
+  targetId: string;
+}
+
+interface SkyBrowserUpdate
+  extends Pick<SkyBrowserState, 'selectedBrowserId' | 'cameraInSolarSystem'> {
+  browsers: { [id: string]: SkyBrowserBrowser };
+}
+
+export interface SkyBrowserState {
```

Comment:
I think @engbergandreas and I previously discusse that the redux slices should only have the type for the actual state, and that other types/interfaces should be moved to a types.d.ts. file.  So here I would only keep this one, and move the other types above to another file. 

the motivation is to make the code for the redux slices easier to overview, and make it clearer what information is part of each state in the sotre